packages/react/src/components/StructuredList/StructuredList.js

```diff
@@ -25,11 +25,6 @@ export class StructuredListWrapper extends Component {
      */
     className: PropTypes.string,
 
-    /**
-     * Specify whether a border should be added to your StructuredListWrapper
-     */
-    border: PropTypes.bool,
```

Comment:
I think we might need to wrap this in a `deprecate` block as lame as that is otherwise if someone is passing in `border` it might get spread onto `<section>` with `other` accidentally :disappointed: This could cause React to complain about the extra attribute showing up.

Reply:
😞

Reply:
Do we need to keep the rest of the `border` code as well? Or is keeping the prop definition wrapped in a deprecate enough?

Reply:
@tw15egan I doubt we need to keep the class name code. As long as we have:

```js
    const { children, selection, className, ariaLabel, border: _border, ...other } = this.props;
```

I think it'd be fine so it doesn't get caught up in `other`.  The rest of the code can totally be dropped I think (alongside the deprecate bit for the prop type)

Reply:
Alright I've updated it, let me know if the changes look good 👍